packages/core/src/modules/oob/domain/OutOfBandDidCommService.ts

```diff
@@ -35,6 +36,24 @@ export class OutOfBandDidCommService extends DidDocumentService {
   @IsString({ each: true })
   @IsOptional()
   public accept?: string[]
+
+  public get resolvedDidCommService(): ResolvedDidCommService {
+    return {
+      id: this.id,
+      recipientKeys: this.recipientKeys.map((didKey) => DidKey.fromDid(didKey).key),
+      routingKeys: this.routingKeys?.map((didKey) => DidKey.fromDid(didKey).key) ?? [],
+      serviceEndpoint: this.serviceEndpoint,
+    }
+  }
+
+  public static fromResolvedDidCommService(service: ResolvedDidCommService) {
+    return new OutOfBandDidCommService({
+      id: service.id,
+      recipientKeys: service.recipientKeys.map((key) => new DidKey(key).did),
+      routingKeys: service.routingKeys.map((key) => new DidKey(key).did),
+      serviceEndpoint: service.serviceEndpoint,
+    })
+  }
```

Comment:
Don't think it is incorrect, but it seems a bit "misleading" that the recipientKeys in the first object references keys and the second one dids. Would it make sense to rename something? If not then we shouldn't change it to make sense, but it felt a bit strange to me.